packages/plugin/src/documents.ts

```diff
@@ -15,13 +15,18 @@ const handleVirtualPath = (documents: Source[]): Source[] => {
   return documents.map(source => {
     const location = source.location!;
     if (['.gql', '.graphql'].some(extension => location.endsWith(extension))) {
-      return source;
+      return {
+        ...source,
+        // When using glob pattern e.g. `**/*.gql` location contains always forward slashes even on
+        // Windows
+        location: path.resolve(location),
+      };
```

Comment:
@yoavain-sundaysky this is better fix, so other rules will get always os specific slash